crates/solver/src/arguments.rs

```diff
@@ -240,6 +240,12 @@ pub struct Arguments {
     #[clap(long, env, use_value_delimiter = true)]
     pub transaction_submission_nodes: Vec<Url>,
 
+    /// Don't submit high revert risk (i.e. transactions that interact with on-chain
+    /// AMMs) to the public mempool. This can be enabled to avoid MEV when private
+    /// transaction submission strategies are available.
+    #[clap(long, env)]
```

Comment:
Seems like we mostly use
```rust
    #[clap(long, env, parse(try_from_str), default_value = "false")]
```
for boolean CLI arguments.

Reply:
This was from a time when we needed to do this for `kontemplate` to allow it to be parameterized based on some template value. Note that we no longer have this restriction (as we can now encode this as conditionals and do it else where).

Personally, I find the "there or not there" flags more intuitive and "Unix-y". I don't feel super strongly, so I am OK with changing it (but, AFAIU, we currently do a mix of flags and `parse(try_from_str), default_value = "false"`).

Maybe as a follow-up I can change this to be more consistent.

Reply:
I also prefer to not pass `true` or `false` I just think the last time this thing came up we opted for passing the values explicitly. I think the follow-up PR to make everything consistent would be nice. 👌

Reply:
> I think the follow-up PR to make everything consistent would be nice. 

I actually found a bug with the `--enable-blockscout` flag in doing this PR 🤣. Good thing we did it!